windows/qt-build-base/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM microsoft/windowsservercore:1709
+LABEL maintainer=bain@accupara.com description="accupara qt build container"
+
+
+RUN @powershell -NoProfile -ExecutionPolicy Bypass -Command \
+    $Env:chocolateyVersion = '0.10.8' ; \
+    $Env:chocolateyUseWindowsCompression = 'false' ; \
+    "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+RUN choco install openssh --force -y -params '"/SSHServerFeature /KeyBasedAuthenticationFeature"'
```

Comment:
Can we convert the multiple RUN into a single RUN command please:
```
RUN choco install openssh --force -y -params '"/SSHServerFeature /KeyBasedAuthenticationFeature"' && \
     choco install -y rsync \
     etc...
```
At least in the Linux world, this results in less layers and therefore potentially smaller image sizes.
Thanks,